bin/index.js

```diff
@@ -2,7 +2,6 @@
 /* global error */
 'use strict';
 
-require('shelljs/global');
```

Comment:
I'm open to other solutions that don't pollute the global namespace. Feel free to create another PR if you'd like to see that changed.